skypy/galaxy/tests/test_mass.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import scipy.stats
+import scipy.integrate
+import scipy.special as sc
+import pytest
+
+import skypy.galaxy.mass as mass
+import skypy.utils.special as special
+
+
+def test_exponential_distribution():
+    # When alpha=0, M*=1 and x_min~0 we get a truncated exponential
+    q_max = 1e2
+    sample = mass.peng_masses(0, 0, 1, size=1000,
+                              x_min=1e-10, x_max=q_max,
+                              resolution=1000)
+    d, p_value = scipy.stats.kstest(sample, 'truncexpon', args=(q_max,))
+    assert p_value >= 0.01
+
+
+def test_peng_masses():
+    # Test that error is returned if redshift input is an array but size !=
+    # None and size != redshift,size
+    with pytest.raises(ValueError):
```

Comment:
This test is failing and I don't understand why. It's basically the same code as in the case of sampling the luminosities.